### DISCUSSION TITLE
Monthly Cost Report of all Resources with tags and EA Account + Subscription Info

### DISCUSSION THREAD

**Discussion by AndTraHTI** [mivano/azure-cost-cli#118] [Ideas]
@mivano I create a json file of all resources under our Enterprise Agreement with the command attached. This works well so far, but I would like to add the account and the subscription of the resources. Is this possible?
azure-cost dailyCosts -b xxxxxxxx --timeframe BillingMonthToDate --dimension ResourceGroupName --includeTags -o json

**Comment by mivano** [comment]
Yeah that is an interesting one. Originally, it only supported a subscription id, so the output was always scoped to a subscription and it did not make sense to fetch and output the subscription id as well (as they would all be the same). With the new options for billing accounts, this is no longer implicit, so I do agree that it needs to be included somewhere. I probably need to include it in the filter request.

**Reply by AndTraHTI** [reply]
Great, I'm glad you like the suggestion. I can't wait for a possible implementation!

**Comment by AndTraHTI** [comment]
@mivano I have also noted when using the azure-cost dailyCosts -e option to specify an account only the costs of one subscription are exported.
